unit-tests-java8/src/test/java/org/eclipse/collections/test/map/mutable/MutableMapTestCase.java

```diff
@@ -53,4 +61,36 @@ default void Iterable_remove()
                 this.getExpectedFiltered(3, 3, 3, 2, 1),
                 this.getExpectedFiltered(3, 3, 2, 2, 1)));
     }
+
+    @Test
+    default void Iterable_computeIfPresent()
+    {
+        MutableMap<Integer, Integer> map = this.newWithKeysValues(1, 1, 2, 2, 3, 3);
+        Map<Integer, Integer> hashMap = new LinkedHashMap<>();
+        hashMap.put(1, 1);
+        hashMap.put(2, 2);
+        hashMap.put(3, 3);
+        if (map instanceof UnmodifiableMutableMap)
```

Comment:
Instead of an instanceof, we can use inheritance. This ought to be similar to how MutableMapIterableTestCase#Map_clear is overridden in UnmodifiableMutableMapTest#Map_clear